examples/glueviz_wcs_example.ipynb

```diff
@@ -0,0 +1,208 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename_d1 = 'NIRISS_f150w_direct_dit1_nis_cal.fits'\n",
+    "filename_d2 = 'NIRISS_f150w_direct_dit2_nis_cal.fits'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with warnings.catch_warnings():\n",
+    "    warnings.simplefilter('ignore')  # Silence warnings\n",
+    "    dither1 = load_data(filename_d1)[0]  # 0 = SCI\n",
+    "    dither2 = load_data(filename_d2)[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Shows the data collection.\n",
+    "dither1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Shows the data components for SCI extension.\n",
+    "dither1.components"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we create a `glue-jupyter` app and add the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app = jglue()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.add_data(dither1)\n",
+    "app.add_data(dither2);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Having loaded both SCI extensions from different dithers and know their WCS components, we can now link them.\n",
+    "\n",
+    "**TODO: What does this really do? Even with this added, when I pan in first viewer, the second one does not auto-pan.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.add_link(dither1, 'Right Ascension', dither2, 'Right Ascension')\n",
+    "app.add_link(dither1, 'Declination', dither2, 'Declination')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We display both dithers in separate, but connected, viewers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "viewer1 = app.imshow(data=dither1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "viewer2 = app.imshow(data=dither2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**TODO: Do we need this?**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# DOES NOT WORK!\n",
+    "# http://glueviz.org/glue-jupyter/developer_notes.html\n",
+    "\n",
+    "#from glue_jupyter.link import link as gjlink\n",
+    "#gjlink((viewer1.state, 'visible'), (viewer2.state, 'visible'))"
```

Comment:
http://glueviz.org/glue-jupyter/developer_notes.html has some notes about using bi-directional links with `link(...)` function. But the examples given are some custom viewers that I am not interested in writing. I cannot get this to work. But I am not even sure if this is needed. Won't the data linking above be sufficient? Do I have to link so many different things?